tests/Unit/Service/Bulk/BulkUpdateUsersAssignmentsStateServiceTest.php

```diff
@@ -76,6 +77,7 @@ public function testItAddsBulkOperationFailureIfWrongOperationTypeReceived(): vo
         );
 
         $expectedAssignment = (new Assignment())
+            ->setId(new UuidV6('00000001-0000-6000-0000-000000000000'))
```

Comment:
you could isolate UUID string into a variable and remove these string duplications

Reply:
I don't think that adds any value here to be honest.